omniport/core/public/index.html

```diff
@@ -40,6 +40,8 @@
   <!-- Load Semantic UI Toasts assets from the backend -->
   <link rel="stylesheet" type="text/css" href="/static/react-semantic-toasts/react-semantic-alert.css" />
 
+<!-- Load Pure React Carousel assets from the backend -->
+<link rel="stylesheet" type="text/css" href="/static/pure-react-carousel/react-carousel.es.css" />  
```

Comment:
Add a line break after this

Reply:
Also, if one file is used here, why 2 files are added in https://github.com/IMGIITRoorkee/omniport-backend/pull/140

Reply:
> Also, if one file is used here, why 2 files are added in [IMGIITRoorkee/omniport-backend#140](https://github.com/IMGIITRoorkee/omniport-backend/pull/140)

the second file is map file to that css file so without map file some code of css doesnot get to frontend.
thats why only one link is required but both files are required at backend